mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -60,8 +62,12 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ElasticSearchIntegrationTest.class);
 
+    private TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
This setting is still supported using the support client, but no more supported by the Embedded ES.

I deleted these lines in the next version of the code.